### DISCUSSION TITLE
Last update broke my Deconz setup with HA - MQTT user changed to "addon" ?

### DISCUSSION THREAD

**Discussion by den-mac** [Koenkk/zigbee2mqtt#21335] [General]
I recently dd an HA update of the zigbee2mqtt add on and all my devices stopped working. I use a Deconz interface for the setup.

I was a bit confused so I started digging around in the logs and whatever else I could find and after looking at the config for the add on I saw that under the MQTT section the mqtt user had changed to "addon" which is not what I set up.

Once I changed it back to my user + password and restarted the add on things started working again.

Anyone else run into this?

**Comment by Koenkk** [comment]
Dupe of https://github.com/Koenkk/zigbee2mqtt/issues/21162
